querydsl-sql/src/test/java/com/querydsl/sql/ConfigurationTest.java

```diff
@@ -68,9 +78,51 @@ public void get_schema() {
         assertEquals("pub", configuration.getOverride(new SchemaAndTable("public", "")).getSchema());
         assertEquals("emp", configuration.getOverride(new SchemaAndTable("", "employee")).getTable());
         assertEquals("employees", configuration.getOverride(new SchemaAndTable("public", "employee")).getTable());
-//        assertEquals("pub", configuration.getSchema("public"));
-//        assertEquals("emp", configuration.getTable("", "employee"));
-//        assertEquals("employees", configuration.getTable("public", "employee"));
+
+        configuration.setDynamicNameMapping(new PreConfiguredNameMapping());
+        SchemaAndTable notOverriddenSchemaAndTable = new SchemaAndTable("notoverridden", "notoverridden");
+        assertEquals(notOverriddenSchemaAndTable, configuration.getOverride(notOverriddenSchemaAndTable));
+
+        configuration.setDynamicNameMapping(new ChangeLetterCaseNameMapping(LetterCase.UPPER, Locale.getDefault()));
```

Comment:
I think using `Locale.getDefault()` is not right here.
then it's the same as just calling `.toLowerCase()`/`.toUpperCase()`.